lib/internal/http2/core.js

```diff
@@ -1492,6 +1507,8 @@ class Http2Stream extends Duplex {
 
     const handle = this[kHandle];
     const req = new WriteWrap();
+    req.timeout = setTimeout(onSlowRead.bind(req, this),
+                             this.session[kState].slowReadTimeout);
```

Comment:
I would prefer that we avoid creating a new timeout for every request and then clear it, as it is going to be very costly. Something like: `if (now - timeLastByteSend >= timeout) slowRead()`?